### PR TITLE
[CodeQuality] Register IssetOnPropertyObjectToPropertyExistsRector to code-quality config set

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ public function refactor(Node $node): ?Node
 
     private function shouldSkipForeach(Foreach_ $foreach): bool
     {
-        if (isset($foreach->keyVar)) {
+        if (property_exists($foreach, 'keyVar') && $foreach->keyVar !== null) {
```
Comment (TomasVotruba) [inline rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: The `$foreach->keyVar` always exists - https://github.com/nikic/PHP-Parser/blob/1d1bc8a36459d67acef508b952dad097960b110e/lib/PhpParser/Node/Stmt/Foreach_.php#L12



```suggestion
        if ($foreach->keyVar !== null) {
```

Comment (TomasVotruba) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: Could you update the `IssetOnPropertyObjectToPropertyExistsRector` so it handle this case too?

Comment (samsonasik) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: Changing that will make the following fixture code not changed as always exists as well:

```php
class SomeClass
{
    private $x;
    private $y;

    public function run()
    {
        isset($this->x);
        isset($this->y);
        isset($this->x) && isset($this->y);
    }
}
```

Do you have any suggestion?

Comment (TomasVotruba) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: I think such case should be changed like this:

```diff
-isset($this->x);
+$thix->x !== null;
```

Comment (samsonasik) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: ok, I will try

Comment (TomasVotruba) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php:142]: It's a rough guess... testing on Rector's code will tell more :)

Comment (samsonasik) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php]: implemented.

Comment (samsonasik) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php]: I think for object (no ThisType) that always has a property need to be changed to `$obj->var !== null` as well for consistency.

Comment (samsonasik) [reply rules/code-quality/src/Rector/Foreach_/ForeachToInArrayRector.php]: implemented.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**